README.md

```diff
@@ -917,6 +917,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Hassan Tahir](https://github.com/thehassantahir)
   - [Hebron Panam Praise](https://github.com/panam-py)
   - [Heetesh Kumar Pradhan](https://github.com/HeeteshSimon)
+  - [Hello Faizan](https://github.com/hellofaizan)
```

Comment:
I am really confused with your PR. Why you added "hello faizan" here? Let's see what others say.